SPACING.md

```diff
@@ -0,0 +1,335 @@
+    private var isSomething: Bool
+
+    /// Documentation
+    private var documentedProperty: String
+
+    /// Documentation
+    private var anotherDocumentedProperty: String
+}
+```
+
+### Property Observers & Getters/Setters
+- use __no__ blank lines at all between closures
+
+```swift
+var name: String {
+    willSet {
+        loadViewIfNeeded()
+    }
+    didSet {
+        nameLabel?.text = name
+    }
+}
+```
+
+### Overridden Functions
+- include __one__ blank line _after_ calling the super method if you add _multiple_ lines to your method
+- include __no__ blank line _after_ calling the super method if you only add _one_ line
+
+```swift
+override func viewWillAppear(_ animated: Bool) {
+    super.viewWillAppear(animated)
+
+    if let parent = parent {
+      // code
+    }    
+}
+
+override func viewDidAppear(_ animated: Bool) {
+    super.viewDidAppear(animated)
+    trackPageView()
+}
+```
+
+### Inside Protocol Definitions (and Enums or Structs without implementations)
+
+- __one__ blank line between the protocol definition and its first property/method, as well as between every property/method signature and (the documentation of) the following one
+- __one__ space between a property's type and every used item in `{ get set }`, `{ get }` or `{ set }`
+
+```swift
+protocol SomeDelegate: class {
+
+    /// Documentation
+    var property: Type { get set }
+
+    /// Documentation
+    func methodOne()
+
+    /// Documentation
+    func methodTwo()
+}
+
+struct Address {
+
+    /// Documentation
+    var street: String
+
+    /// Documentation
+    var city: String
+}
+```
+
+- if no documentation is required, use __no__ blank lines at all
+
+```swift
+enum Side {
+    case left
+    case right
+}
+```
+
+__Exception:__ Structs or enums declared _inside_ classes may occupy too much vertical space so that the blank lines could be omitted (however one might argue that such nested types should be declared in a separate extension anyway)
+
+```swift
+class BasketViewController: UIViewController {
+
+    fileprivate enum State {
+        /// An error occured while loading the basket.
+        case error
+        /// A loading operation that is related to the whole view is in progress.
+        case loading
+        /// The basket does not contain any products.
+        case emptyBasket
+        /// The basket contains products and should be displayed.
+        case filledBasket
+        /// There is currently no logged in user.
+        case notAuthenticated
+    }
+}
+```
+
+
+### Marks
+Marks should always be surrounded by __exactly one__ blank line above and below.
+
+```swift
+      ...
+      return something
+    }
+}
+
+// MARK: - SomeDelegate
+
+extension ViewController: SomeDelegate {
+
+    func delegateMethod() {}
+}
+```
+
+### Inside Methods and other Blocks of Code
+- __do not__ start with a blank line (exceptions may exist where adding __one__ blank line at the beginning might increase readability)
```

Comment:
Yup, this is a reasonable idea, although honestly comes a bit too strong for my taste – I would say 

__Generally do not start with a blank line__, unless this helps readability.